### PR TITLE
add missing XZ dep in easyconfig for libunwind 1.1 w/ GCC/4.9.2, sync sanity_check_paths across libunwind easyconfigs

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2005.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/82c3eb6fadad260d1806d7cb9bd676e7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2413.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1e91327a341c82eedb462d093667ab2f for a full test report.

File: easybuild/easyconfigs/l/libunwind/libunwind-1.1-GCC-4.9.2.eb [hunk 1]
```diff
@@ -15,6 +15,16 @@ toolchain = {'name': 'GCC', 'version': '4.9.2'}
 sources = [SOURCE_TAR_GZ]
 source_urls = [GNU_SAVANNAH_SOURCE]
 
+checksums = [
+    'fb4ea2f6fbbe45bf032cd36e586883ce',     # libunwind-1.1.tar.gz
```
Comment (wpoely86) [inline easybuild/easyconfigs/l/libunwind/libunwind-1.1-GCC-4.9.2.eb]: I thought we didn't split for single entries?

Comment (boegel) [inline easybuild/easyconfigs/l/libunwind/libunwind-1.1-GCC-4.9.2.eb]: I'm happy to collapse this if you feel strongly about it.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2005.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3884983c1849185e960ed678ec531e57 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**